Closes #449

wwojciech opened the pull request:
Closes #449

danielinteractive commented on NEWS.md:
```diff
@@ -1,4 +1,5 @@
 # Version 1.0.9000.9115
+* Added new method `sampleSize-Samples`.
```
```suggestion
* Added method `sampleSize()` for objects of class `Samples`.
```

wwojciech commented:
Hi @danielinteractive - I renamed `sampleSize` to `size` now and also adjusted (and deleted a generic `size`) for the `CohortSize` class. I am wondering if you would like to review it before I merge it, as there are many changes.
